roles/openshift_node/tasks/main.yml

```diff
@@ -23,7 +23,6 @@
       iptables_sync_period: "{{ openshift_node_iptables_sync_period | default(None) }}"
       kubelet_args: "{{ openshift_node_kubelet_args | default(None) }}"
       labels: "{{ lookup('oo_option', 'openshift_node_labels') | default( openshift_node_labels | default(none), true) }}"
-      portal_net: "{{ openshift_master_portal_net | default(None) }}"
```

Comment:
Is the whole portal_net needed, or just the DNS IP? I'm fine with persisting either though, I just didn't see a need to keep persisting a value on the node that was not needed.